screens/Navigation/ProfileNav.tsx

```diff
@@ -23,53 +13,13 @@ export const ProfileScreenConfig = {
 };
 
 export default function ProfileNav() {
-  const router = useRouter();
-
-  const colorScheme = useColorScheme();
-  const account = useCurrentAccount();
-  const options: NativeStackNavigationOptions = {
-    headerTintColor:
-      Platform.OS === "android"
-        ? textSecondaryColor(colorScheme)
-        : textPrimaryColor(colorScheme),
-    animation: navigationAnimation,
-  };
-
   return (
     <NativeStack.Screen
       name="Profile"
       component={ProfileScreen}
-      options={({ route }) => ({
-        presentation: "modal",
-        headerTitle: "",
-        headerLeft: () => (
-          <ScreenHeaderModalCloseButton onPress={router.goBack} />
-        ),
-        headerRight: () => {
-          if (route.params.address === account) {
-            if (Platform.OS === "ios") {
-              return (
-                <ScreenHeaderButton
-                  title={translate("modify")}
-                  onPress={() => {
-                    navigate("UserProfile");
-                  }}
-                />
-              );
-            }
-            return (
-              <ScreenHeaderIconButton
-                iconName="square.and.pencil"
-                onPress={() => {
-                  navigate("UserProfile");
-                }}
-              />
-            );
-          }
-          return null;
-        },
-        ...options,
-      })}
+      options={{
+        title: "Profile",
+      }}
```

Comment:
Should we use `useHeader` hook and use our header 🤔  not sure haven't look at the screen but feels like it